### PR TITLE
define postgres db as variable

### PR DESCRIPTION
**Pull request by pgvishnuram**
currently postgres db values are hardcoded inside module , this change introduced to move from hardcoded value to variable type.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -117,6 +117,12 @@ variable "deploy_db" {
   description = "Do you want a database deployed in this project?"
 }
 
+variable "db_version" {
+  type        = string
+  default     = "POSTGRES_9_6"
```
Comment (danielhoherd) [inline variables.tf:122]: Can we add a comment here with a hyperlink to GCP docs listing available versions? That would be really helpful to anybody who is using this module.

**Comment by pgvishnuram** [comment]
```bash
variable "db_version" {
  type        = string
  default     = "POSTGRES_9_6"
  description = "define postgres database version. Full list is https://cloud.google.com/sql/docs/postgres/db-versions ( '.' and spaces to be replaced with '_' while defining in values  ex: PostgreSQL 9.6 as POSTGRES_9_6) " 
}
```

File: variables.tf (hunk 1 above)
Comment (danielhoherd) [reply variables.tf:122]:
```suggestion
  # https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance#argument-reference
  default     = "POSTGRES_9_6"
```

**Comment by pgvishnuram** [comment]
@danielhoherd - we can merge this to master

**Comment by pgvishnuram** [comment]
@danielhoherd after PR60 merge pls merge this as well

**Pull request merged**